tests/integration/datadictwithobjid/submission/utils.py

```diff
@@ -12,7 +12,7 @@
 re_md5 = re.compile(r"(i?)(?<![a-z0-9])[a-f0-9]{32}(?![a-z0-9])")
 
 data_fnames = [
-    "experiment.json",
+    "experiment.1.json",
```

Comment:
looks like `experiment.2.json` and `experimental_metadata.json` are unused - either add tests or delete the json files